documentation/src/main/asciidoc/introduction/Configuration.adoc

```diff
@@ -543,3 +543,31 @@ On the other hand, if only _some_ columns store nationalized data, use the link:
 Alternatively, you can configure SQL Server to use the UTF-8 enabled collation `_UTF8`.
 ====
 
+[[datetime-jdbc]]
+=== Date and time types and JDBC
+
+By default, Hibernate handles date and time types defined by `java.time` by:
+
+- converting `java.time` types to JDBC date/time types defined in `java.sql` when sending data to the database, and
+- reading `java.sql` types from JDBC and then converting them to `java.time` types when retrieving data from the database.
+
+This works best when the database server time zone agrees with JVM system time zone.
+
+TIP: We therefore recommend setting things up so that the database server and the JVM agree on the same time zone. **Hint:** when in doubt, UTC quite a nice time zone.
```

Comment:
```suggestion
TIP: We therefore recommend setting things up so that the database server and the JVM agree on the same time zone. **Hint:** when in doubt, UTC is quite a nice time zone.
```